rubocop-cobra/Gemfile.lock

```diff
@@ -49,7 +49,7 @@ GEM
     pry-byebug (3.9.0)
       byebug (~> 11.0)
       pry (~> 0.13.0)
-    rack (2.2.4)
+    rack (3.0.0)
```

Comment:
major version upgrade 👀

Reply:
I wouldn't be too concerned about this one specifically. It's introduced by rubocop-rails, but this is rubocop, so I don't even know why they would need rack. Also, it doesn't change the dependency, only the resolved dependency in development/test (Gemfile.lock).